client/components/Achievements.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Game from './Game.jsx';
+import NavigationBar from './NavBar.jsx';
+import { getPurchases, progressBar, average, previousMonth } from '../helpers/gameHelpers.jsx';
```

Comment:
Be prepared for some work refactoring this as we move your functions to the backend!

It shouldn't be too cray though. We will end up rendering achievements based on a few of their characteristics passed down from the server.